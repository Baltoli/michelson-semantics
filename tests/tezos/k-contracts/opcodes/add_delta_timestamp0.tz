$contract { parameter (pair int timestamp);
storage (option timestamp);
code { CAR; DUP; CAR; DIP{CDR}; ADD; SOME; NIL operation; PAIR}
 } ;
$param (Pair 100 100) ;
$storage None ;
$ExpectedOutput (Some "1970-01-01T00:03:20Z") ;

