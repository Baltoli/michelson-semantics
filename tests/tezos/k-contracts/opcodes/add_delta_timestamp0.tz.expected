<generatedTop>
  <k>
    .
  </k>
  <stack>
    Pair .List Some #Timestamp ( 200 )
  </stack>
  <paramtype>
    pair .AnnotationList int .AnnotationList timestamp .AnnotationList
  </paramtype>
  <storagetype>
    option .AnnotationList timestamp .AnnotationList
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( "InvalidMyAddr" )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( "InvalidSourceAddr" )
  </sourceaddr>
  <senderaddr>
    #Address ( "InvalidSenderAddr" )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
